### PR TITLE
daemon/api_quotas.go: include current memory usage information in results

### PR DESCRIPTION
**Pull request by anonymouse64**
This will enable us to report the memory-usage information in the command line output.

### CODE REVIEW

**Review by bboozzoo — APPROVED** [review]
LGTM

File: systemd/systemd.go
Comment (bboozzoo) [inline systemd/systemd.go:606]: Not having to update the tests may indicate that we're missing a bit of coverage here. Should be fairly trivial to add a test for this branch though.

Comment (anonymouse64) [reply systemd/systemd.go:606]: Added a test case for this

**Comment by anonymouse64** [comment]
There seems to be a real issue here with Amazon Linux 2: https://pastebin.ubuntu.com/p/5HtkVjHkV8/

I'm investigating, hopefully it's nothing too problematic

**Review by stolowski — APPROVED** [review]
Nice, thank you!

File: daemon/api_quotas.go
Comment (anonymouse64) [reply daemon/api_quotas.go:60]: As discussed, in a followup before making this non-experimental we will revise the API to have a `constraints` and `current` section instead to match the CLI output

File: daemon/api_quotas.go
Comment (anonymouse64) [reply daemon/api_quotas.go:100]: Yeah a bulk query would make sense

**Comment by anonymouse64** [comment]
This is blocked until #10329 lands, which fixes CentOS and Amazon Linux 2 support.

**Comment by anonymouse64** [comment]
the ordering of these pr's has gotten too muddled so I will close these and reopen them in a more logical splitting/ordering
but to be clear, there is now a functional spread test on master we can use/modify to demonstrate whether changes are working or not

**Pull request closed without merging**
